resources/scripts/syncTrustStore.groovy

```diff
@@ -1,4 +1,4 @@
-import com.sonatype.nexus.ssl.plugin.internal.CertificateRetriever
+import org.sonatype.nexus.ssl.CertificateRetriever
```

Comment:
@jtognazzi, which version of `org.sonatype.nexus.plugins:nexus-ssl-plugin` are you using in your `build.gradle`? I don't find this class in that package. Look at the latest version of the `nexus-ssl-plugin` at https://repository.sonatype.org/content/groups/sonatype-public-grid/org/sonatype/nexus/plugins/nexus-ssl-plugin/3.40.1-01/

Reply:
I did not change the build.gradle the nexus-ssl-plugin is verions.nxrm3 => 3.23.0-03

Maybe I should have ? 
For info, if your search for [CertificateRetriever](https://repository.sonatype.org/#nexus-search;classname~CertificateRetriever) you'll find it in 
`org.sonatype.nexus nexus-ssl-3.40.1-01.jar`
but not anymore in 
`org.sonatype.nexus.plugin nexus-ssl-plugin for version > 3.38.1-01`

I got the error with nexus 3.39.0-01